.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -500,31 +505,79 @@ spec:
       '*': '*'
   options:
     max_session_ttl: 4m0s
-version: v3
+version: v5
 ```
+
+Create a user that has no access to anything but allows you to request roles:
 ```
 kind: role
 metadata:
-  name: default
+  name: test-role-based-requests
 spec:
   allow:
     request:
       roles:
-      - allow-roles
-      - allow-users
+      - allow-roles-and-nodes
+      - allow-users-with-short-ttl
       suggested_reviewers:
       - random-user-1
       - random-user-2
-  options:
-    max_session_ttl: 8h0m0s
-version: v3
+version: v5
 ```
-- [ ] Verify that under requestable roles, only `allow-roles` and `allow-users` are listed
-- [ ] Verify input validation requires at least one role to be selected
+
+- [ ] Verify that under requestable roles, only `allow-roles-and-nodes` and `allow-users-with-short-ttl` are listed
 - [ ] Verify you can select/input/modify reviewers
-- [ ] Verify after creating a request, requests are listed in pending states
+- [ ] Verify you can view the request you created from request list (should be in pending states)
+- [ ] Verify there is list of reviewers you selected (empty list if none selected AND suggested_reviewers wasn't defined)
 - [ ] Verify you can't review own requests
 
+### Creating Access Requests (Search Based)
+Create a role with access to searcheable resources (apps, db, kubes, nodes, desktops). The template `searcheable-resources` is below.
```

Comment:
Might be worth noting that this is an enterprise feature and won't work with OSS.